src/Compilers/Core/Portable/Symbols/TypeCompareKind.cs

```diff
@@ -38,5 +38,7 @@ internal enum TypeCompareKind
         AllIgnoreOptionsForVB = IgnoreCustomModifiersAndArraySizesAndLowerBounds | IgnoreTupleNames,
 
         CLRSignatureCompareOptions = TypeCompareKind.AllIgnoreOptions & ~TypeCompareKind.IgnoreCustomModifiersAndArraySizesAndLowerBounds,
+
+        AllIgnoreOptionsPlusNullableWithUnknownMatchesAny = TypeCompareKind.AllIgnoreOptions & ~(TypeCompareKind.IgnoreNullableModifiersForReferenceTypes),
```

Comment:
>WithUnknownMatchesAny [](http://example.com/codeflow?start=36&length=21)

Does "WithUnknownMatchesAny" refer to effect of ```ObliviousNullableModifierMatchesAny``` flag? Consider using the same terminology then (and the one we are using for many years now), "WithObliviousMatchesAny"

Reply:
I kept the original name from where I moved this combination of options. I don't love the name either, but that's beyond the purpose for this PR